interfaces/Portalicious/src/app/services/log.service.ts

```diff
@@ -0,0 +1,113 @@
+import { Injectable } from '@angular/core';
+import { DebugPlugin } from '@microsoft/applicationinsights-debugplugin-js';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { environment } from 'src/environments/environment';
+
+// All properties sorted alphabetically.
+export enum LogEvent {
+  userLogout = 'user/logout',
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LogService {
+  private appInsights: ApplicationInsights;
+  private appInsightsInitiated: boolean;
```

Comment:
```suggestion
  private appInsightsInitialized: boolean;
```

Sorry to be that guy but initiated means something very different :D

Reply:
Ha. _Thanks_ Copilot-autocomplete. 😶